rust/plugin/src/helpers/branch.rs

```diff
@@ -85,6 +85,6 @@ pub struct BranchState {
     pub merge_info: Option<BranchStateMergeInfo>,
 	pub revert_info: Option<BranchStateRevertInfo>,
     pub is_main: bool,
-	pub created_by: Option<String>,
-	pub merged_into: Option<DocumentId>,
+	// pub created_by: Option<String>,
+	// pub merged_into: Option<DocumentId>,
```

Comment:
This too; This is for the option to hide the branch if it has already been merged